compiler/resolution/functionResolution.cpp

```diff
@@ -3947,7 +3945,7 @@ static void handleTaskIntentArgs(CallInfo& info, FnSymbol* taskFn) {
       // records and strings are (incorrectly) captured at the point
       // when the task function/arg bundle is created.
       if (taskFn->hasFlag(FLAG_COBEGIN_OR_COFORALL) == true &&
-          isConstValWillNotChange(varActual)        == false &&
+          !varActual->isConstValWillNotChange_AnyIntent()   &&
```

Comment:
We should try and keep the style here uniform ( '!' vs. '== false').

Reply:
I don't see much benefit at this time for the "_AnyIntent" method. I'd prefer rolling back to the 'isConstValWillNotChange' standalone function.

Reply:
@benharsh - I would like to use the "_AnyIntent" method in my upcoming forall-intents PR. Seems you'd be OK to keep it? We can surely tune the names.

Reply:
In that case let's keep it for now, and think of better names later.